src/commands/createVirtualMachine/PassphrasePromptStep.ts

```diff
@@ -0,0 +1,55 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { AzureNameStep, IAzureNamingRules, IWizardOptions } from "vscode-azureextensionui";
+import { ext } from "../../extensionVariables";
+import { localize } from "../../localize";
+import { ConfirmPassphraseStep } from "./ConfirmPassphraseStep";
+import { IVirtualMachineWizardContext } from "./IVirtualMachineWizardContext";
+
+export const passphraseNamingRules: IAzureNamingRules = {
```

Comment:
Yeah this type of stuff is for `AzureNameStep`, which doesn't apply here. Just define the min down in `validatePassphrase`